models/shipping-method/src/shipping-method/types.ts

```diff
@@ -0,0 +1,24 @@
+import type {
+  ShippingMethod,
+  ShippingMethodDraft,
+  TaxCategory,
+} from '@commercetools/platform-sdk';
+import type { TBuilder } from '@commercetools-test-data/core';
+
+export type TShippingMethod = ShippingMethod & { taxCategory: TaxCategory };
+export type TShippingMethodRest = ShippingMethod;
+export type TShippingMethodGraphql = TShippingMethod & {
+  __typename: 'ShippingMethod';
```

Comment:
General question for these types, do we need to add the fields here that we are adding in the transformers?